### DISCUSSION TITLE
🧩 FEATURE REQUEST: Community Leaderboard

### DISCUSSION THREAD

**Discussion by bobbyiliev** [dailydotdev/daily#1016] [Feature Requests]
### Concise Description

Implementation of a Community Leaderboard

### Description

The Daily.Dev community is growing stronger by the day 💪 

One feature that I believe could potentially enhance the user experience and engagement within the community is the addition of a leaderboard based on the reputation points system.

### Solution Proposed

A leaderboard could serve as a means of recognizing and rewarding active and helpful community members. It could foster a sense of healthy competition, encouraging members to participate more actively in discussions, share insights, and assist others.

### Additional Context

```bash
_No response_
```

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @bobbyiliev,

Thanks for yet another suggestion. 🤩
We've discussed it several times but have yet to prioritize it.
I hope with the team's growth, we will be able to achieve much more 💪
